H2O-3/introduction-to-machine-learning-with-h2o-3-automl/introduction-to-machine-learning-with-h2o-3-automl.md

```diff
@@ -1,576 +1,938 @@
+~~~
+
+![aml-cl-leaderboard-2](assets/aml-cl-leaderboard-2.png)
+
+
+By looking at the leaderboard, we can see that the best model is the Stacked Ensemble with all the models, meaning that this Stacked Ensemble was built using all the models that were trained. Secondly, we have the Stacked Ensemble with the best models from each family. The Best of Family Ensemble will usually have a GLM, a Distributed Random Forest, Extremely-Randomized Forest, a GBM, and XGBoost, and Deep Learning model if you give it enough time to train all those models.
+
+Now, let’s see how the best model performs on our test set.
+
+~~~python
+aml.leader.model_performance(test_data=test)
+~~~
+
+![automl-cl-perf-1](assets/automl-cl-perf-1.png)
+![automl-cl-perf-2](assets/automl-cl-perf-2.png)
+
+By looking at the results, we can see that in fifteen minutes, and with less data, AutoML obtained scores close to what we obtained in the first tutorial. The AUC that we obtained was **0.824.** Although this is could be a good AUC, because we have a very imbalanced dataset, we must also look at the misclassification errors for both classes. As you can see, our model is having a hard time classifying bad loans; this is mainly due because only about 3.6% of loans are labeled as bad loans. However, the model is doing very well when classifying good loans; although it is still far from being the best model, this gives us a solid starting point.
+
+We can also take a quick look at the ROC curve:
+
+~~~python
+%matplotlib inline
+aml.leader.model_performance(test_data=test).plot()
+~~~
+![automl-cl-roc](assets/automl-cl-roc.png)
+
+As you can see, it’s the same AUC value that we obtained in the model summary, but the plot helps us visualize it better.
+
+Lastly, let’s make some predictions on our test set.
+
+~~~python
+aml.predict(test)
+~~~
+
+![automl-cl-preds](assets/automl-cl-preds.png)
+
+As we mentioned in the first tutorial, the predictions we get are based on a probability. In the frame above, we have a probability for **FALSE,**, and another one for **TRUE**. The prediction, **predict**, is based on the threshold that maximizes the F1 score. For example, the threshold that maximizes the F1 is about `0.1061`, meaning that if the probability of **TRUE** is greater than the threshold, the predicted label would be **TRUE.**
+
+After exploring the results for our classification problem, let’s use AutoML to explore a regression use-case.
+
+**Task 4 - R Version**
+
+We already have our train and test sets, so we just need to choose our response variable, as well as the predictors. We will do the same thing that we did for the first tutorial.
+
+~~~r
+# Identify predictors and response for the classification use case
+ignore <- c("DELINQUENT", "PREPAID", "PREPAYMENT_PENALTY_MORTGAGE_FLAG", "PRODUCT_TYPE")
+y <- "DELINQUENT"
+
+x <- setdiff(colnames(train), ignore)
+~~~
+
+Now we are ready to run AutoML. Below you can see some of the default parameters that we could change for AutoML
```

Comment:
add a colon at the end of this sentence: "Now we are ready to run AutoML. Below you can see some of the default parameters that we could change for AutoML"